File: codigos algoritmicos javascript/ejercicio29.js
console.log("cliente normal o cliente vip");

entradanormal = 100;
entradavip = 200;
clientepaga = 99;

if (clientepaga >= entradanormal) {

    if (clientepaga >= entradavip) {
        console.log("Entrada VIP: descuento del 20%");
    } else {
        console.log("Entrada normal: descuento del 5%");
    }

} else {
    console.log("No alcanza para ninguna entrada");
}

File: codigos algoritmicos python/ejercicio29.py
print("Cliente normal o cliente VIP?")

entradanormal = 100
entradavip = 200

entrada = int(input("Ingrese valor de la entrada a comprar: "))

if entrada >= entradavip:
    print("Usted obtiene una entrada VIP y un descuento del 20%")

else:

    if entrada >= entradanormal:
        print("Usted obtiene una entrada normal y un descuento del 5%")

    else:
        print("Valor de entrada no válido")